samza-sql/src/main/java/org/apache/samza/sql/interfaces/SqlSystemSourceConfig.java

```diff
@@ -47,30 +48,34 @@
   private final SystemStream systemStream;
 
   private final String source;
-  private String relSchemaProviderName;
+  private final String relSchemaProviderName;
 
-  private Config config;
+  private final Config config;
 
-  private List<String> sourceParts;
+  private final List<String> sourceParts;
+
+  private final boolean isTable;
+
+  private final TableDescriptor tableDescriptor;
 
   public SqlSystemSourceConfig(String systemName, String streamName, Config systemConfig) {
```

Comment:
I think we probably might need to rename this as SQLIOConfig

Reply:
will do